moldesign/_tests/test_pdbfixer_xface.py

```diff
@@ -91,6 +92,22 @@ def test_mutation_nomenclature_string_only(pdb3aid):
     assert len(mut25) == 2
     assert [r.resname for r in mut25] == ['MET', 'MET']
 
+def test_mutation_topology():
+    """ Test the topology of the backbone atoms for a mutated molecule. """
+    molecule = mdt.read(get_data_path('1yu8.pdb'))
+    mutation_residues = ["X.13G"]
+    mutated_molecule = mdt.mutate_residues(molecule, mutation_residues)
+    # Check that the number of bonds for backbone atoms match.
+    backbone_atoms = mdt.data.BACKBONES["protein"]
+    for res, mut_res in zip(molecule.residues, mutated_molecule.residues):
+        for atom in res.atoms:
```

Comment:
FYI, you can directly get an iterator over backbone atoms with `res.backbone`

Reply:
Of course you can!

I will change the code.